ledger-service/http-json/src/main/scala/com/digitalasset/http/util/LedgerOffsetUtil.scala

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.http.util
+
+import com.digitalasset.ledger.api.v1.ledger_offset.LedgerOffset
+import scalaz.{-\/, \/, \/-}
+
+object LedgerOffsetUtil {
+
+  private val LongEitherLongLongOrdering: Ordering[Long \/ (Long, Long)] = {
+    import scalaz.std.tuple._
+    import scalaz.std.anyVal._
+    val O = implicitly[scalaz.Order[Long \/ (Long, Long)]]
```

Comment:
```suggestion
    val O = scalaz.Order[Long \/ (Long, Long)]
```

Explicit implicit resolutions of scalaz typeclasses should always use the apply method, avoiding `implicitly`.

Reply:
it compiles... don't really care any more.

Reply:
but yeah the change makes sense... did not know that there is a `scalaz.Order` object available, was implicitly resolving the trait :)